scripts/gulp/styles.js

```diff
@@ -1,66 +1,104 @@
+    } ) )
+    .pipe(plugins.rename({
       basename: 'capital-framework'
     }))
     .pipe(gulp.dest('./dist'))
-    .pipe($.cssmin())
-    .pipe($.rename({
+    .pipe(plugins.cssmin())
+    .pipe(plugins.rename({
       suffix: '.min'
     }))
     .pipe(gulp.dest('./dist'));
-} );
+}
 
-// Compile all the individual component files so that users can `npm install`
-// a single component if they desire.
-gulp.task( 'styles:components', function() {
+/**
+ * Compile all the individual component files so that users can `npm install`
+ * a single component if they desire.
+ * @returns {PassThrough} A source stream.
+ */
+function stylesComponents() {
   return gulp.src('./src/' + (component || '*') + '/src/*.less')
-    .pipe($.ignore.exclude(function(vf) {
+    .pipe(plugins.ignore.exclude(function(vf) {
       // Exclude Less files that don't share the same name as the directory
       // they're in. This filters out things like cf-vars.less but still
       // includes cf-core.less.
       var matches = vf.path.match(/\/([\w-]*)\/src\/([\w-]*)\.less/);
       // We also exclude cf-grid. It needs its own special task. See below.
       return matches[2] === 'cf-grid' || matches[1] !== matches[2];
     }))
-    .pipe($.less({
+    .pipe(plugins.less({
       paths: ['node_modules/cf-*/src/']
     }))
-    .pipe($.rename(function (path) {
+    .pipe( plugins.autoprefixer( {
+      browsers: [ 'last 2 version',
+                  'not ie < 7',
+                  'android 4',
+                  'BlackBerry 7',
+                  'BlackBerry 10' ]
+    } ) )
+    .pipe(plugins.rename(function (path) {
       path.dirname = component || path.dirname;
       path.dirname = path.dirname.replace('/src','');
     }))
     .pipe(gulp.dest('./tmp'))
-    .pipe($.cssmin())
-    .pipe($.rename({
+    .pipe(plugins.cssmin())
+    .pipe(plugins.rename({
       suffix: '.min'
     }))
     .pipe(gulp.dest('./tmp'));
-} );
+}
 
-// cf-grid needs to compile cf-grid-generated.less
-gulp.task( 'styles:grid', function() {
+/**
+ * cf-grid needs to compile cf-grid-generated.less.
+ * @returns {PassThrough} A source stream.
+ */
+function stylesGrid() {
   return gulp.src('./src/cf-grid/src-generated/*.less')
-    .pipe($.less({
+    .pipe(plugins.less({
       paths: ['node_modules/cf-*/src/']
     }))
-    .pipe($.rename({
+    .pipe( plugins.autoprefixer( {
+      browsers: [ 'last 2 version',
+                  'not ie < 7',
+                  'android 4',
+                  'BlackBerry 7',
+                  'BlackBerry 10' ]
+    } ) )
+    .pipe(plugins.rename({
       basename: 'cf-grid'
     }))
     .pipe(gulp.dest('./tmp/cf-grid'))
-    .pipe($.cssmin())
-    .pipe($.rename({
+    .pipe(plugins.cssmin())
+    .pipe(plugins.rename({
       suffix: '.min'
     }))
     .pipe(gulp.dest('./tmp/cf-grid'));
-} );
+}
+
+gulp.task( 'styles:cf', stylesCf );
+gulp.task( 'styles:components', stylesComponents );
+gulp.task( 'styles:grid', stylesGrid );
```

Comment:
I think so. It groups all the tasks at the bottom of the file always. But out of scope for this PR.